### DISCUSSION TITLE
Can’t open link to PDF file in Github

### DISCUSSION THREAD

**Discussion by user1man** [community/community#50469] [Mobile]
I have found a link and a repository where I am unable to open the PDF, and the link cannot be opened on a mobile client. The only solution is to open the current page in Safari and then open the link next to it. The page with the link is:
[https://github.com/UberGuidoZ/Flipper#if-youre-new-to-github-in-general-check-out-this-great-guide-from-wr3nch](https://github.com/UberGuidoZ/Flipper#if-youre-new-to-github-in-general-check-out-this-great-guide-from-wr3nch)
[Pdf link](https://github.com/wrenchathome/flipperfiles/blob/main/_Guides/How2Flipper.pdf)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.105.0 (94356885)
Device: iPhone 11 (iOS 16.4)
</details>

**Comment by Iron-Ham** [comment]
Hi @user1man,

Thanks for reporting. I was able to reproduce this on iOS and identify the source of the bug. I'll create an internal bug report from this discussion to track the issue. 

Hesham
